README.md

```diff
@@ -1,2 +1,27 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+Attempts = 5
+Random_Words = 10
+word_length = 5
+Answer == 1 Random_Words
+
+
+Likeness
+	Number of correct letters in choosen word
```

Comment:
The worksheet asks for pseudocode to calculate the number of correct letters, which you have not provided. Be sure to read the specification carefully in future!